hack/source-manifests.sh

```diff
@@ -49,18 +49,23 @@ fi
 # ==== DUMP NOOBAA YAMLS ====
 noobaa_dump_crds_cmd="crd yaml"
 noobaa_dump_csv_cmd="olm csv yaml"
+echo "Dumping Noobaa csv using command: $IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_csv_cmd"
 ($IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_csv_cmd) > $NOOBAA_CSV
+echo "Dumping Noobaa crds using command: $IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_crds_cmd"
 ($IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_crds_cmd) > $OUTDIR_CRDS/noobaa-crd.yaml
```

Comment:
It would also help if you used `tee` for these commands to show the operator output as well as store it to a file, i.e. something like:
```
($IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_csv_cmd) | tee $NOOBAA_CSV
($IMAGE_RUN_CMD $NOOBAA_IMAGE $noobaa_dump_crds_cmd) | tee $OUTDIR_CRDS/noobaa-crd.yaml 
```

Reply:
good idea, i made a PR. https://github.com/openshift/ocs-operator/pull/182